rsts/getting_started.rst

```diff
@@ -44,15 +44,15 @@ Make sure you have `Git <https://git-scm.com/>`__, and `Python <https://www.pyth
 
 Start a new project / repository
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Install Flyte's Python SDK — `Flytekit <https://pypi.org/project/flytekit/>`__ (recommended using a  `virtual environment <https://docs.python.org/3/library/venv.html>`__) and clone the `flytekit-python-template <https://github.com/flyteorg/flytekit-python-template>`__ repo.
+Install Flyte's Python SDK — `Flytekit <https://pypi.org/project/flytekit/>`__ (recommended using a  `virtual environment <https://docs.python.org/3/library/venv.html>`__), run a command to initialize your project, that command will prompt you for the name of directory where your example workflows are going to live. The ``init`` command comes with a help menu that can be shown by running ``pyflyte init --help``. For this guide we're going to use ``myflyteapp`` in the initial invocation.
 
 .. prompt:: bash (venv)$
 
     pip3 install flytekit --upgrade
-    git clone https://github.com/flyteorg/flytekit-python-template.git myflyteapp
+    pyflyte init myflyteapp
```

Comment:
here is where we can add a comment saying, `myflyteapp` or your own app name?

Reply:
should we also say, it uses cookie-cutter underneath?

Reply:
the paragraph right above mentions the choice of `myflyteapp`. I'll make it more explicit that it's actually part of the invocation (and that the user is free to pick whatever name they want).

As for letting them know that it's using cookie-cutter, do you really think it's needed? From a beginner user perspective they shouldn't need to know that it uses this or that templating engine. WDYT?

Reply:
We can trust that the users that want to know what's going on under the covers will find out about cookie-cutter from the cli itself:
```
❯ pyflyte init --help
...
Usage: pyflyte init [OPTIONS] PROJECT_NAME

  Create flyte-ready projects.

Options:
  --template TEXT  cookiecutter template folder name to be used in the repo -
                   https://github.com/flyteorg/flytekit-python-template.git

  --help           Show this message and exit.
```